### PR TITLE
Edit updated contributing template.

### PR DESCRIPTION
**Pull request by batpigandme**
_n.b._ I made the changes directly to the template, and held off on re-generating the usethis-specific contributing guide.

### CODE REVIEW

**Comment by hadley** [comment]
Looks good - can you please run `use_tidy_contributing()` on usethis itself?  (`load_all()` won't work unless you first do `devtools::unload("devtools")`)

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -8,20 +8,20 @@ For more detailed info about contributing to this, and other tidyverse packages,
...
 
 If you want to make a bigger change, it's a good idea to first file an issue and make sure someone from the team agrees that it’s needed. 
-If you’ve found a bug, please an issue that illustrates the bug with a minimal 
+If you’ve found a bug, please file an issue that illustrates the bug with a minimal 
 [reprex](https://www.tidyverse.org/help/#reprex) (this will also help you write a unit test, if needed).
 
 ### Pull request process
 
-*   Fork the package and clone on to your computer. If you haven't done this before, we recommend using `usethis::create_from_github("r-lib/usethis", fork = TRUE)`.
+*   Fork the package and clone onto your computer. If you haven't done this before, we recommend using `usethis::create_from_github("batpigandme/usethis", fork = TRUE)`.
```
Comment (jennybc) [inline .github/CONTRIBUTING.md]: this should remain `r-lib/usethis`

Comment (batpigandme) [reply .github/CONTRIBUTING.md]: (>ლ) fixed. Thanks, Jenny.

**Comment by hadley** [comment]
Thanks!

**Pull request merged**